helpers/isLessonID.tsx

```diff
@@ -0,0 +1,4 @@
+import { LessonID } from '../@types/lesson'
+export default function isLessonID(arg: LessonID | string): arg is LessonID {
+  return ['1', '2', '3', '4', '5', '24', '25', '27', '28', '29'].includes(arg)
```

Comment:
Lesson Ids should not be hardcoded, because if any lesson are added in the future, this file will have to be updated.